data/ka'het missions.txt

```diff
@@ -0,0 +1,138 @@
+	on offer
+		event "graveyard label"
+		fail
+
+event "graveyard label"
+	galaxy "label graveyard"
+		sprite label/graveyard
+
+mission "Disabled Ka'het"
+	boarding
+	invisible
+	source
+		government "Ka'het" " Ka'het "
+	destination "Earth"
+	on offer
+		log "Successfully boarded a Ka'het, it appears to be a single, ship-sized organism that can travel in space using a special exoskeleton."
+		conversation
+			`Your ship slowly approaches the <origin>, now floating lifeless near you. As you move closer, you spot a massive hole in the hull of the ship, revealing what appears to be a large, slug-like being. While most of the ship's machinery doesn't seem to have received heavy damages, your weapons must have hit the slug's vital organs, killing it, after which point the ship seems to have stopped working.`
+			`	You have no troubles docking with what seems to be its "exoskeleton", trying to decide what to do next: while the Ka'het is too different from a ship for you to command it, it's safe to say that you can now loot its outfits.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1A"
+	landing
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "Graveyard Label: offered"
+		not "remnant met taely"
+	on offer
+		conversation
+			`Do you want to ask the Remnant about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You ask around more than a dozen people, but none of them seems to be aware of what you're talking about. Either the information is too confidential to be given to a stranger like you, or you didn't ask the right people.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1B"
+	landing
+	invisible
+	source "Viminal"
+	to offer
+		has "Graveyard Label: offered"
+		has "remnant met taely"
+	on offer
+		conversation
+			`Do you want to ask Taely about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You find Taely inside one of the outer hangars of the shipyard, apparently teaching a small group of people how to make basic repairs on an Albatross. You sit on a bench waiting for her to finish her lesson, after which she walks over to you. "Hello, <first>!" she says. "Is there something I can help you with?" `
+			`	"I'm here because I need to ask you a few questions," you reply.`
+			`	As she sits near you, you start explaining her about the area you reached, and the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```

Comment:
```suggestion
			`	As she sits near you, you start explaining to her the area you reached, including the weird alien ships that attacked you with weapons similar to the ones the Remnant possess. It doesn't seem to be news for her. "We know of the existence of these Ka'het, the aliens you encountered, since we first reached the Ember Wastes. We found many of their 'shells' in our planets, crashed there thousands or even tens of thousands of years before our arrival, but with some tech so advanced to still be conserved enough to be reverse engineered.`
```
Technically the player could get this mission without having been attacked, but I'll let it slide.